ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/claimantresponse/ClaimantResponseMapper.java

```diff
@@ -55,16 +58,16 @@ public CCDClaimantResponse to(Claim claim) {
                 .ifPresent(rejection::freeMediationOption);
             responseRejection.getAmountPaid().ifPresent(rejection::amountPaid);
             responseRejection.getReason().ifPresent(rejection::reason);
+            claim.getClaimantRespondedAt().ifPresent(rejection::submittedOn);
             return rejection.build();
         }
         throw new MappingException("unsupported claimant response type " + claimantResponse.getType());
     }
 
-    public Claim from(CCDClaimantResponse ccdClaimantResponse) {
+    public void from(CCDClaimantResponse ccdClaimantResponse,Claim.ClaimBuilder claimBuilder) {
```

Comment:
please format your code, spacing is right. we follow IntelliJ default formatting

Reply:
same as above